samples/.net/endpoints/batch/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿// <copyright file="Program.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+using Azure.Core;
+using Azure.Identity;
+using Azure.ResourceManager.MachineLearning;
+
+namespace BatchInferencingSamples
+{
+    internal class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            TokenCredential credentials = new DefaultAzureCredential();
+            
+            Guid subscriptionId = Guid.Parse("<your_subscription_id>");
+            string resourceGroupName = "<your_resource_group_name>";
+            string workspaceName = "<your_workspace_name>";
+            string batchEndpointName = "<your_endpoint_name>";
```

Comment:
Should we parse args for these?

Reply:
hmmm we can iterate on the args parsing..!